test/plugin.test.js

```diff
@@ -337,3 +337,21 @@ QUnit.test('custom error details should override defaults', function(assert) {
   assert.strictEqual(document.querySelector('.vjs-errors-message').textContent,
     customError.message, 'message should match custom override value');
 });
+
+QUnit.test('Append Flash error details when flash is not supported', function(assert) {
+  let isFlashSupported = videojs.getComponent('Flash').isSupported();
+
+  if (!isFlashSupported) {
```

Comment:
I see what you are trying to say now. 
Also how the mock-up will need a change to the `if statement.`
Implementing the changes.